src/utils/ReadOnlyException.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { InternalException } from './InternalException';
+
+export class ReadOnlyException extends InternalException {
```

Comment:
`Exception` is more of a C# word AFAIK and `InternalException` just hasn't been renamed properly yet. Let's use `Error` moving forward

Reply:
Renamed to `Error`